### PR TITLE
[infra] Revise docker to use python3 for pip3

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise docker to use python3 for pip3 command.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

File: infra/docker/bionic/Dockerfile [hunk 1]
```diff
@@ -43,8 +43,8 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive \
     apt-get -qqy install doxygen graphviz wget zip unzip clang-format-8 python3 python3-pip python3-venv hdf5-tools pylint curl
 RUN apt-get update && apt-get -qqy install python3.8 python3.8-venv
-RUN pip3 install --upgrade pip
-RUN pip3 install yapf==0.22.0 numpy
+RUN python3.8 -m pip install --upgrade pip
+RUN python3.8 -m pip install yapf==0.22.0 numpy
```
Comment (seanshpark) [inline infra/docker/bionic/Dockerfile:47]: need to use pyhon3.8 but there is no pip3.8

File: infra/docker/focal/Dockerfile [hunk 2]
```diff
@@ -35,8 +35,8 @@ RUN apt-get update && apt-get -qqy install libprotobuf-dev protobuf-compiler
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive \
     apt-get -qqy install doxygen graphviz wget zip unzip clang-format-8 python3 python3-pip python3-venv hdf5-tools pylint curl
-RUN pip3 install --upgrade pip
-RUN pip3 install yapf==0.22.0 numpy
+RUN python3 -m pip install --upgrade pip
+RUN python3 -m pip install yapf==0.22.0 numpy
```
Comment (seanshpark) [inline infra/docker/focal/Dockerfile:39]: to follow bionic

File: infra/docker/bionic/Dockerfile (hunk 1 above)
Comment (hseok-oh) [reply infra/docker/bionic/Dockerfile:47]: I need to check `yapf` working on format checker.

**Comment by hseok-oh** [comment]
@seanshpark Isn't it better to change default `python3` to `python3.8` by using `update-alternatives`?

File: infra/docker/bionic/Dockerfile (hunk 1 above)
Comment (hseok-oh) [reply infra/docker/bionic/Dockerfile:47]: I confirmed `yapf` works.

**Review by hseok-oh — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
> Isn't it better to change default python3 to python3.8 by using update-alternatives?

I hesitate to ask or request users to do this, as users may want to use default 3.6 version in Ubuntu 18.04.
I assume our CI would be better to test with OS default settings and use it's own alternative methods to use different version.

**Pull request merged**